### PR TITLE
Add couchdb security docs

### PR DESCRIPTION
**Pull request by maxlath**
CouchDB databases access control is set from [security documents](http://docs.couchdb.org/en/1.6.1/api/database/security.html) in which admins and members should be defined.

From the CouchDB web interface:

> Database admins can update design documents and edit the admin and member lists.
> Database members can access the database. If no members are defined, the database is public.

This PR just automates setting `CONFIG.store.username` as the only admin and member for each databases. Aiming to solve #11

### CODE REVIEW

File: db/lib/put_security_doc.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  const username = CONFIG.store.username
+  if (!_.isNonEmptyString(username)) {
+    throw new Error(`bad CONFIG.db.username: ${username}`)
+  }
+  return {
+    // Database admins can update design documents
+    // and edit the admin and member lists.
+    admins: { names: [username] },
+    // Database members can access the database.
+    // If no members are defined, the database is public.
```
Comment (almereyda) [inline db/lib/put_security_doc.js:29]: This is interesting.

File: db/lib/put_security_doc.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+const CONFIG = require('config')
+const __ = CONFIG.universalPath
+const _ = __.require('lib', 'utils')
+const breq = require('bluereq')
+const dbUrl = CONFIG.store.url()
+
+module.exports = function (db, dbName) {
+  return db.get('_security')
+  .spread(function (body, headers) {
```
Comment (almereyda) [inline db/lib/put_security_doc.js:9]: Straightforward. Also see http://bluebirdjs.com/docs/api/spread.html

**Comment by almereyda** [comment]
Cool, thanks, let me have a look at this before merging.

**Comment by almereyda** [comment]
It seems nano doesn't make use of #18

**Comment by maxlath** [comment]
made obsolete by #30

**Pull request closed without merging**